src/components/ProgressButton/ProgressButton.jsx

```diff
@@ -13,25 +13,36 @@ class ProgressButton extends React.Component {
 
   render() {
     const beforeText = this.props.beforeText ? (
-      <span className="button-icon">{this.props.beforeText} </span>
+      <span className="button-icon" aria-hidden="true">
+        {this.props.beforeText}
+        &nbsp;
```

Comment:
Adding a non-breaking space to prevent text & arrow getting separated by a new line. I've seen this in some screenshots shared by designers, but I wasn't able to duplicate it.

Reply:
Can you post one of those screenshots from designers?

Reply:
A form start button with a lot of text could wrap on mobile screens
![Screen Shot 2021-10-04 at 12 08 06 PM](https://user-images.githubusercontent.com/136959/135894377-ecfeebdf-cb91-4830-891c-de3b2b6c27c6.png)

Adding a non-breaking space, forces it to wrap
![Screen Shot 2021-10-04 at 12 09 22 PM](https://user-images.githubusercontent.com/136959/135894451-1530bc14-ce5f-4d8a-b028-3e5082b3c740.png)